packages/thirdweb/src/wallets/utils/chains.ts

```diff
@@ -27,7 +27,7 @@ export function getValidPublicRPCUrl(chain: ChainMetadata) {
  * Get valid RPCs for given chain
  * @internal
  */
-function getValidChainRPCs(
+export function getValidChainRPCs(
```

Comment:
likely don't want to expose this publicly, we should work towards not having / needing / using this function